src/components/ProHalfCourt/CustomiseWindow.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Box,
+  Button,
+  Collapse,
+  Flex,
+  FormControl,
+  FormLabel,
+  Icon,
+  Input,
+  Spacer,
+  useDisclosure,
+  Text,
+  ButtonGroup,
+  FormHelperText,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { BiChevronDown, BiChevronUp, BiPencil } from "react-icons/bi";
+// inputWidth={clipWidth} setInputWidth={setClipWidth} inputLength = {clipLength} setInputLength={setClipLength
+const CustomiseWindow = ({ setInputWidth, setInputLength }: any) => {
+  const { isOpen, onToggle } = useDisclosure();
+
+  const [widthValue, setWidthValue] = useState("");
+  const [hightValue, setHightValue] = useState("");
```

Comment:
1. The type of the property should be changed to number
2. The property name is hightValue, but it corresponds to the length, please be consistent

Reply:
I fix that.